python/revng/daemon/graphql.py

```diff
@@ -0,0 +1,198 @@
+#
```

Comment:
Is this just a rename or actual new code?
GitHub shows it as new code.

Reply:
It was renamed from `static_handlers.py`, but since a lot of code has been dropped from it Github hasn't detected it as a rename